src/Utils.ts

```diff
@@ -0,0 +1,4 @@
+export const groupClassNames = (groupName: string = '', classes: string[] = []) =>
```

Comment:
@rinick @slavamuravey Let me know if there is a better place for this method.

Reply:
Let's place utils this file, I don't mind.

But, I suggest to remove concatenation of other classes, because it is not the function responsibility.
The library already has https://www.npmjs.com/package/classnames dependency, so, we should not do its work.
The function will look like:

```
const groupClassNames = (groupNames: string = '') => groupNames
  .split(' ')
  .filter(value => value !== '')
  .map((name) => `dock-style-${name}`);
```

Reply:
oh... didn't realize the project had `classnames`. Sure, will fix this.